source/options.css

```diff
@@ -1,6 +1,12 @@
+@media not screen and (width: 400px) {
+	/* Excludes Edge */
```

Comment:
You moved the comment. It should be next to the query because that’s what it’s about

Reply:
Sorry vs code formatter messed it up. Will fix